docs/using.md

```diff
@@ -15,85 +15,93 @@
    [Kaniko](https://github.com/GoogleContainerTools/kaniko)) and others will be
    specific to your project (e.g. running your particular set of unit tests).
 2. Create a `Pipeline` which expresses the Tasks you would like to run and what
-   [Resources](#creating-resources) the Tasks need. Use
-   [`providedBy`](#providedBy) to express the order the `Tasks` should run in.
+   [PipelineResources](#resources-in-a-pipeline) the Tasks need. Use
+   [`providedBy`](#providedBy) to express when the input of a `Task` should come
+   from the output of a previous `Task`.
 
 See [the example Pipeline](../examples/pipeline.yaml).
 
+### PipelineResources in a Pipeline
+
+In order for a `Pipeline` to execute, it will probably need
+[`PipelineResources`](#creating-pipelineresources) which will be provided to
+`Tasks` as inputs and outputs.
+
+Your `Pipeline` must declare the `PipelineResources` it needs in a `resources`
+section in the `spec`, giving each a name which will be used to refer to these
+`PipelineResources` in the `Tasks`.
+
+For example:
+
+```yaml
+spec:
+  resources:
+  - name: my-repo
+    type: git
+  - name: my-image
+    type: image
+```
+
+These `PipelineResources` can then be provided to `Task`s in the `Pipeline` as
+inputs and outputs, for example:
+
+```yaml
+spec:
+  #...
+  tasks:
+  - name: build-the-image
+    taskRef:
+      name: build-push
+    resources:
+      inputs:
+      - name: workspace
+        resource: my-repo
+      outputs:
+      - name: image
+        resource: my-image
+```
+
 ### ProvidedBy
 
-When you need to execute `Tasks` in a particular order, it will likely be
-because they are operating over the same `Resources` (e.g. your unit test Task
-must run first against your git repo, then you build an image from that repo,
-then you run integration tests against that image).
+Sometimes you will have `Tasks` that need to take as input the output of a previous
+`Task`, for example, an image built by a previous `Task`.
 
-We express this ordering by adding `providedBy` on `Resources` that our `Tasks`
+We express this dependency by adding `providedBy` on `Resources` that our `Tasks`
```

Comment:
"This dependency is expressed by …" (if we want to avoid the use of `we` – #k8s-docs-guidelines)

Reply:
whoops, this is gonna be a hard habit to break for ~us~ me :sweat_smile:

Reply:
maybe "Express this dependency by adding..." - avoiding passive voice? :D